README.md

```diff
@@ -145,6 +145,9 @@ git clone https://github.com/ggerganov/llama.cpp
 cd llama.cpp
 make
 
+#For Windows and CMake, use the following command instead:
+cmake -S . -B build/ -G "MinGW Makefiles"
```

Comment:
this only works for mingw. but installing a vs community edition is much simpler for average users.

Reply:
Yes indeed, you could add the following lines for the people using vs and cmake:

```
cd <path_to_llama_folder>
mkdir build
cd build
cmake ..
cmake --build . --config Release
```

Reply:
Thanks, I have updated it with your suggestion